recipe/fix-customize-build.patch

```diff
@@ -0,0 +1,74 @@
+diff --git a/imagecodecs/libjpeg.pxd b/imagecodecs/libjpeg.pxd
+index 501d468..45af300 100644
+--- a/imagecodecs/libjpeg.pxd
++++ b/imagecodecs/libjpeg.pxd
+@@ -47,9 +47,9 @@ cdef extern from 'jpeglib.h' nogil:
+     struct jpeg_error_mgr:
+         int msg_code
+         const char** jpeg_message_table
+-        noreturn_t error_exit(jpeg_common_struct*)
+-        void output_message(jpeg_common_struct*)
+-        void format_message(jpeg_common_struct* cinfo, char* buffer)
++        noreturn_t error_exit(jpeg_common_struct*) nogil
++        void output_message(jpeg_common_struct*) nogil
++        void format_message(jpeg_common_struct* cinfo, char* buffer) nogil
+ 
+     struct jpeg_common_struct:
+         jpeg_error_mgr* err
+diff --git a/setup.py b/setup.py
+index f0e4f7a..3c3ac0c 100644
+--- a/setup.py
++++ b/setup.py
+@@ -518,6 +518,7 @@ def customize_build_condaforge(EXTENSIONS, OPTIONS):
+     # del EXTENSIONS['htj2k']
+     del EXTENSIONS['jetraw']
+     del EXTENSIONS['jpegxs']
++    del EXTENSIONS['jpegxl']
+     # del EXTENSIONS['jpegli']
+     del EXTENSIONS['lzfse']
+     del EXTENSIONS['lzham']
+@@ -527,9 +528,9 @@ def customize_build_condaforge(EXTENSIONS, OPTIONS):
+     del EXTENSIONS['sperr']
+     del EXTENSIONS['sz3']
+     del EXTENSIONS['ultrahdr']
+-    del EXTENSIONS['zlibng']
++    del EXTENSIONS['blosc2']  # c-blosc2 library not commonly available
+ 
+-    EXTENSIONS['jpeg8']['sources'] = []  # use libjpeg-turbo 3
++    #EXTENSIONS['jpeg8']['sources'] = []  # use libjpeg-turbo 3

```

Comment:
Nitpick - might be beneficial to leave a comment here why this is commented out.

Reply:
I left a comment in `meta.yaml` about all the fixes that the patch is doing